main.js

```diff
@@ -18,4 +19,285 @@ function close() {
+        </button>
+        
+    </footer>
+
+</div>
+
+<div class="popup_articlediv show2">
+
+    <header class="popup_flex1">
+
+        <h3><a class="textdecoration font1" href="#">${cardsData[btnIndex].name[0]}</a></h3>
+
+        <button type="button" class="close_popup"><img src="images/close.png" alt=""></button>
+
+    </header> 
+
+    <ul class="list1 ul1">
+
+        <li><a class="textdecoration font2" href="#">CANOPY</a></li>
+        <li><a class="textdecoration" href="#"><img src="media/mid_dot.jpg" alt=""></a></li>
+        <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
+        <li><a class="textdecoration" href="#"><img src="media/mid_dot.jpg" alt=""></a></li>
+        <li><a class="textdecoration font3" href="#">2015</a></li>
+
+    </ul>
+
+    <figure class="figure2 ">
+
+        <a href="#"><img src="${cardsData[btnIndex].image[1]}" alt="Nature_img"></a>
+
+    </figure>
+
+    <article class="popup_flex2">
+        <p class="width80">${cardsData[btnIndex].description[1]}</p>
+
+        <div>
+
+            <ul class="gridbox2">
+
+                <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologiesPop[0]}</a></li>
+                <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologiesPop[1]}</a></li>
+                <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologiesPop[2]}</a></li>
+                <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologiesPop[3]}</a></li>
+                <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologiesPop[4]}</a></li>
+                <li class="background1"><a class="textdecoration font4" href="#">${cardsData[btnIndex].technologiesPop[5]}</a></li>
+
+            </ul>
+
+            <footer>
+                <button class="align_3 footer_btn textdecoration" id="see-projects-btn" type="submit">
+                <a href="${cardsData[btnIndex].liveLink}"><span>See Live</span></a>
+                    <img src="images/live.png" alt="">
+                </button>
+
+                <button class="align_3 footer_btn textdecoration" id="see-projects-btn" type="submit">
+                <a href="${cardsData[btnIndex].Source}"><span>See Source</span></a>
+                    <img src="images/github.jpg" alt="">
+                </button>
+            </footer>
+            
+        </div>
+       
+    </article>
+
+</div>
+
+</article>
+`;
+
+    const body = document.querySelector('body');
+    body.appendChild(popUpContainer);
+    const popUpBox = document.querySelector('.popup_section');
+    popUpBox.innerHTML = popupEle;
+    const closePopup = document.querySelectorAll('.close_popup');
+    closePopup.forEach((e) => {
+      e.addEventListener('click', () => {
+        popUpContainer.remove();
+      });
+    });
+  });
+});
```

Comment:
- [ ] Kindly make sure you update your linters config as currently you are using the wrong one. **[Here](https://github.com/microverseinc/linters-config/tree/master/html-css-js)** is where you can find all information you need about the new linters configuration